tc/sgx/trusted_worker_manager/enclave/work_order_enclave_kme.cpp

```diff
@@ -0,0 +1,121 @@
+/* Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "enclave_t.h"
+
+#include <string>
+
+#include <sgx_trts.h>
+#include<mbusafecrt.h>
+
+#include "error.h"
+#include "tcf_error.h"
+#include "utils.h"
+
+#include "enclave_utils.h"
+#include "base_enclave.h"
+#include "enclave_data.h"
+#include "work_order_processor.h"
+
+ByteArray last_serialized_response_kme;
```

Comment:
We need to see how to avoid this global variable. This will get overwritten if KME get requests from multiple WPEs (even if WPEs handles same workload type). Currently the work order is executed in serial order. So it might work. Later on we may need to use map of workorder_id as key and serialized_response_kme as value. We can do in a separate PR.